src/types/jobDescription.ts

```diff
@@ -0,0 +1,356 @@
+                <vscode-data-grid-cell grid-column="2">${this.jobDescriptionInfo?.timeSliceEndPool}</vscode-data-grid-cell>
+              </vscode-data-grid-row>
+              <vscode-data-grid-row>
+                <vscode-data-grid-cell grid-column="1">Allow Multiple Threads</vscode-data-grid-cell>
+                <vscode-data-grid-cell grid-column="2">${this.jobDescriptionInfo?.allowMultipleThreads}</vscode-data-grid-cell>
+              </vscode-data-grid-row>
+              <vscode-data-grid-row>
+                <vscode-data-grid-cell grid-column="1">Workload Group</vscode-data-grid-cell>
+                <vscode-data-grid-cell grid-column="2">${this.jobDescriptionInfo?.workloadGroup}</vscode-data-grid-cell>
+              </vscode-data-grid-row>
+              <vscode-data-grid-row>
+                <vscode-data-grid-cell grid-column="1">ASPGRP</vscode-data-grid-cell>
+                <vscode-data-grid-cell grid-column="2">${this.jobDescriptionInfo?.aspgrp}</vscode-data-grid-cell>
+              </vscode-data-grid-row>
+              <vscode-data-grid-row>
+                <vscode-data-grid-cell grid-column="1">DDM Conversation</vscode-data-grid-cell>
+                <vscode-data-grid-cell grid-column="2">${this.jobDescriptionInfo?.ddmConversation}</vscode-data-grid-cell>
+              </vscode-data-grid-row>
+            </vscode-data-grid>
+            `;
+
+        const libraryListTab = `<h1>Library list</h1>
+            <vscode-data-grid>
+              <vscode-data-grid-row row-type="header">
+                <vscode-data-grid-cell cell-type="columnheader" grid-column="1"><b>Sequence</b></vscode-data-grid-cell>
+                <vscode-data-grid-cell cell-type="columnheader" grid-column="2"><b>Library</b></vscode-data-grid-cell>
+              </vscode-data-grid-row>
+              ${this.jobDescriptionLibraries?.map(library => {
+                  return /*html*/`
+                  <vscode-data-grid-row>
+                    <vscode-data-grid-cell grid-column="1">${library.sequence}</vscode-data-grid-cell>
+                    <vscode-data-grid-cell grid-column="2">${library.library}</vscode-data-grid-cell>
+                  </vscode-data-grid-row>`;
+                }).join("")}
+            </vscode-data-grid>
+            `;
+
+        const panels = /*html*/`
+            <vscode-panels>
+                <vscode-panel-tab id="tab-1">
+                  DESCRIPTION
+                </vscode-panel-tab>
+                <vscode-panel-tab id="tab-2">
+                  LIBRARY LIST
+                  <vscode-badge appearance="secondary">${this.jobDescriptionLibraries?.length ? this.jobDescriptionLibraries?.length: 0}</vscode-badge>
+                </vscode-panel-tab>
+                <vscode-panel-view id="view-1">${descriptionTab}</vscode-panel-view>
+                <vscode-panel-view id="view-2">${libraryListTab}</vscode-panel-view>
+            </vscode-panels>`;
```

Comment:
Try using the webview toolkit's `Components.panels` here:
```suggestion
        const panels = Components.panels([
          {title: "DESCRIPTION", content: descriptionTab },
          {title: "LIBRARY LIST", content: libraryListTab, badge: this.jobDescriptionLibraries?.length ? this.jobDescriptionLibraries?.length : 0}
      ]);
```